07-routing-gossip.md

```diff
@@ -798,6 +897,42 @@ and allows bootstrapping for new nodes as well as updating for nodes that
 have been offline for some time.  The `gossip_queries` option
 allows for more refined synchronization.
 
+### Inventory-based Rebroadcasting
+
+With the simple rebroadcasting scheme described above, nodes that are connected to many different peers will receive duplicates for the same gossip messages.
+
+A more efficient scheme based on inventory messages, similar to how transactions and blocks are propagated on the bitcoin p2p network, can optionally be used: if the `option_inv_gossip` feature bit is set, instead of broadcasting `channel_annoucement` and `channel_update` messages, nodes will broadcast a `gossip_inventory` message which is similar to a `reply_channel_range` message and can be used to filter announcements by timestamp or by content (checksum):
+
+1. type: 266 (`gossip_inventory`)
+2. data:
+    * [`32`:`chain_hash`]
+	* [`tlvs`:`gossip_inventory_tlvs`]
+
+1. tlvs: `gossip_inventory_tlvs`
+2. types:
+   1. type: 1 (`channel_inv`)
```

Comment:
is the idea here that this will only include the `short_channel_ids` of new `channel_updates`? and then the receiver can request the `channel_announcement` if it doesn't know of it?

Reply:
Yes that's correct.

Reply:
Gotcha, makes sense to me. No sense announcing a channel_announcement if we don't have any updates for it!